doc/modules/clustering.rst

```diff
@@ -540,6 +546,14 @@ graph, and SpectralClustering is initialized with `affinity='precomputed'`::
    <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.19.8100>`_
    Andrew Y. Ng, Michael I. Jordan, Yair Weiss, 2001
 
+ * `"Robust and efficient multi-way spectral clustering"
```

Comment:
not sure that this all needs to be here. it does need to be in the class docstring

Reply:
I have just added one new reference in the style consistent with all other references around. I have no opinion on appropriateness of the style of the references here. At any rate, I should not be the one changing the style of all these references, if it is justified.